README.md

```diff
@@ -215,7 +215,7 @@ Please see [our helper](https://github.com/sendgrid/sendgrid-go/tree/master/help
 <a name="use-cases"></a>
 # Use Cases
 
-[Examples of common API use cases](https://github.com/sendgrid/sendgrid-go/blob/master/USE_CASES.md), such as how to send an email with a transactional template.
+[Examples of common API use cases](https://github.com/sendgrid/sendgrid-go/blob/master/USAGE.md), such as how to send an email with a transactional template.
```

Comment:
I believe this should actually point to: https://github.com/sendgrid/sendgrid-go/blob/master/use-cases/README.md

Reply:
I actually missed that doc completely so the one that I linked made more sense. Changing it to the one that you suggested.